src/main.ts

```diff
@@ -42,6 +47,9 @@ export async function activate(context: vscode.ExtensionContext): Promise<CSharp
     util.setExtensionPath(extension.extensionPath);
 
     const eventStream = new EventStream();
+    let optionStream = new BehaviorSubject<Options>(Options.Read(vscode));
+    let optionChangedStream = new Subject<void>();
```

Comment:
Can we take this a step further and put the options onto the same message bus as the rest of our notifications?